Remotion/Web/Test.Shared/UpdatePanelTests/SutUserControl.ascx.cs

```diff
@@ -50,14 +51,15 @@ protected override void OnInit (EventArgs e)
       _postBackEventHandler.PostBack += HandlePostBack;
       Controls.Add(_postBackEventHandler);
 
+      var localFallbackUrl = SafeServiceLocator.Current.GetInstance<IFallbackNavigationUrlProvider>().GetURL();
       string asyncPostBackCommandInsideUpdatePanelID = "AsyncPostBackCommandInsideUpdatePanel";
-      AsyncCommandInsideUpdatePanelHyperLink.NavigateUrl = "#";
+      AsyncCommandInsideUpdatePanelHyperLink.NavigateUrl = localFallbackUrl;
```

Comment:
also a test type. here, there's no logic based on this (as opposed to the other test type), so I don't care which direction you choose. Personally, I probably would have left the "#" to reduce complexity.